opendbc/car/honda/carstate.py

```diff
@@ -62,11 +62,6 @@ def get_can_messages(CP, gearbox_msg):
         ("ACC_HUD", 10),
         ("ACC_CONTROL", 50),
       ]
-  else:  # Nidec signals
-    if CP.carFingerprint == CAR.HONDA_ODYSSEY_CHN:
-      messages.append(("CRUISE_PARAMS", 10))
-    else:
-      messages.append(("CRUISE_PARAMS", 50))
```

Comment:
Sometimes signals are added just to do CAN validity checks on them, but I'm not sure if this is important

Reply:
it was used for a "cruise speed offset" (found in commaai/one)